### PR TITLE
Makes swarmers check for mobs in objects

### LINKED ISSUE #27912: Swarmers can kill people
Swarmers can kill people by breaking containers with people inside.

### PR DESCRIPTION
**Pull request by swindly**
:cl: Swindly
fix: Swarmers can no longer deconstruct objects with living things in them.
/:cl:

Fixes https://github.com/tgstation/tgstation/issues/27912

### CODE REVIEW

**Comment by praisenarsie** [comment]
Can they open the locker?

**Comment by swindly** [comment]
no

**Comment by praisenarsie** [comment]
then lockers will become the swarmer's one weakness

**Comment by PKPenguin321** [comment]
really it's an issue with ex_act but whatever

File: code/game/gamemodes/miniantags/bot_swarm/swarmer.dm [hunk 1]
```diff
@@ -402,6 +402,14 @@
...
 	return FALSE
 
+/obj/structure/closet/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	if(!opened)
+		for(var/mob/living/L in contents)
+			if(!issilicon(L) && !isbrain(L))
+				to_chat(S, "<span class='warning'>An organism has been detected inside this object. Aborting.</span>")
+				return FALSE
+	..()
+
```
Comment (coiax) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: This feels like you could probably standardise this for all /obj, just to check for /mob/living in the GetContents(), which would cover human operated machinery and all that.

**Pull request merged**